fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -677,6 +731,7 @@ private void removeEmptyFragments() {
                         throw new RepositoryRuntimeException("Error removing empty fragments", ex);
                     }
                 });
+                touch();
```

Comment:
Actually, if I'm not mistaken, this `touch()` can be removed. The `removeEmptyFragments` method is private and only called twice in FedoraResourceImpl. Each time it is called, the `touch` method is already being called. I will simply remove this.